content/monitors/_index.md

```diff
@@ -68,14 +68,13 @@ Setting the above to **Notify** send an email for the monitor audit events to al
 
 ## Manually resolve your monitor
 
-It only makes sense in a couple cases to manually resolve your monitor:
+The monitor *Resolve* function is artificially switching the monitor status to `OK` for its next evaluation. The following monitor evaluation will be performed normally on the data the monitor is based on.
 
-* If the monitor is in a "no data" state then resolving it hides it from the triggered monitors page.
-* If the monitor is in the triggered state but has stopped reporting data then resolving it hides it from the triggered monitors page.
+If a monitor is alerting because its current data corresponds to its `ALERT` state, *Resolve* will have the monitor to follow the following state switch `ALERT -> OK -> ALERT`. Thus it's not appropriate for acknowledging that you have seen the alert or telling Datadog to ignore the alert.
```

Comment:
If a monitor is alerting because its current data corresponds to its `ALERT` state, *Resolve* will have the monitor follow the state switch `ALERT -> OK -> ALERT`. Thus, it's not appropriate for acknowledging that you have seen the alert or telling Datadog to ignore the alert.